local_settings.ini

```diff
@@ -0,0 +1,11 @@
+# This file contains settings related to the local machine
+[config]
+output_dir=~/simout/quantum-in_situ_opt/skopt
+
+# default parameter file to be used. params-quant_self_opt.ini
+# for ArXiv paper settings use ncl-params-ArXiv1810.12178.ini
```

Comment:
This refers to a file from another project, remove this line.

Reply:
Done.